catroid/src/main/java/org/catrobat/catroid/content/Look.java

```diff
@@ -161,9 +161,6 @@ public Look copyLookForSprite(final Sprite cloneSprite) {
 	}
 
 	public boolean doTouchDown(float x, float y, int pointer) {
-		if (sprite.isPaused) {
-			return true;
-		}
```

Comment:
This is the only real use-case of a paused member.
However, as explained in the commit message, touch events are only fired when the stage is running.
When the stage is paused, a grey overlay covers the stage and captures all touch events.